terraso_backend/tests/graphql/mutations/test_sites.py

```diff
@@ -47,9 +52,21 @@ def test_site_creation(client_query, user):
     assert str(site.id) == id
     assert site.latitude == pytest.approx(site.latitude)
     assert site.longitude == pytest.approx(site.longitude)
+    assert site.owner == user
+
+
+def test_site_creation_in_project(client_query, user, project):
+    kwargs = site_creation_keywords()
+    kwargs["projectId"] = str(project.id)
+    response = client_query(CREATE_SITE_QUERY, variables={"input": kwargs})
+    content = json.loads(response.content)
+    assert "errors" not in content and "errors" not in content["data"]
+    id = content["data"]["addSite"]["site"]["id"]
+    site = Site.objects.get(pk=id)
+    assert site.project == project
 
 
-ADD_CLIENT_QUERY = """
+EDIT_SITE_QUERY = """
```

Comment:
This was just incorrectly named. I changed the variable name so it reflects what the mutation actually does (edits the site).